docs/en_us/platform_api/source/courses/overview.rst

```diff
@@ -0,0 +1,64 @@
+.. _Courses API Overview:
+
+#############################
+Courses API Overview
+#############################
+
+Use the Courses API to obtain information about edX courses.
+
+.. contents::
+   :local:
+   :depth: 1
+
+*****************************************
+Courses API Version and Status
+*****************************************
+
+The Courses API is currently at version 1.0.
+
+************************************
+Courses API Resources and Endpoints
+************************************
+
+The Courses API includes the :ref:`Courses<Courses API Courses Resource>` and
+:ref:`Blocks<Courses API Blocks Resource>` resources, and supports the following
+tasks, methods, and endpoints.
+
+=================
+Courses Resource
+=================
+
+.. list-table::
+   :widths: 20 10 70
+   :header-rows: 1
+
+   * - Task
+     - Method
+     - Endpoint
+   * - :ref:`Get the details for a specified course`
```

Comment:
I expected to see the more general resource /api/courses/v1/courses/ listed first, then the specific /api/courses/v1/courses/{course_key}/. I think that's just my preference based on what I've seen in the past (http://docs.apigee.com/app-services/content/rest-endpoints).